js/index.js

```diff
@@ -26,4 +26,68 @@ for(let i = 0; i < skills.length; i++) {
     skill.innerHTML = skills[i];
     skillList.appendChild(skill);
     skill.style.margin = '0 3em';
-}
\ No newline at end of file
+}
+
+// Creating the Submit Form
+const messageForm = document.querySelector('[name="leave_message"]');
+const leaveMessage = document.querySelector('#leave_message');
+
+messageForm.addEventListener('submit', submitMessage);
+
+function submitMessage(event) {
+    event.preventDefault(); // Prevent the form from submitting the default way
+    
+    const username = event.target.usersName.value;
+    const email = event.target.usersEmail.value;
+    const message = event.target.usersMessage.value;
```

Comment:
I noticed was that depending on the size of the message, the Remove and Edit buttons don't line up across messages. Not a big deal or given in the requirements, but I thought I'd mention it.